packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/link/read.rs

```diff
@@ -108,13 +111,13 @@ async fn by_link_type_by_source_entity_id(
             SELECT link_type_version_id, NULL AS single_link, target_entity_ids AS multi_link
             FROM multi_links
             "#,
-            &[&source_entity_id, link_type_uri, &i64::from(link_type_version)],
+            &[&source_entity_id, &link_type_base_uri.as_str(), &i64::from(link_type_version)],
         )
         .await
         .into_report()
         .change_context(QueryError)
         .attach_printable(source_entity_id)
-        .attach_printable(link_type_uri.clone())?;
+        .attach_printable(link_type_base_uri.clone())?;
```

Comment:
```suggestion
        .attach_printable_lazy(|| link_type_base_uri.clone())?;
```
Otherwise, it would always be cloned, regardless if it's an error or not